### DISCUSSION TITLE
Gopro12 and motion detection

### DISCUSSION THREAD

**Discussion by Ezequiellpi** [gopro/labs#923] [Q&A]
Hello, with any video configuration, the camera always locks after a few minutes and the screen says labs processing. Does anyone have a GoPro 12 with a motion detector working?

**Comment by dnewman-gpsw** [comment]
factory reset as it works here.  However users have reported that it sometimes doesn't continue, that can be fixed by adding !1R in place of !R at the end.

**Reply by Ezequiellpi** [reply]
I've already tried it and nothing. I put it on repeat mode, it makes 20 or 30 videos and it gets stuck...

**Reply by dnewman-gpsw** [reply]
Are you share a video of it being stuck?  Is the camera not responding to other actions?

**Reply by Ezequiellpi** [reply]
Then I try to record a video, but basically the motion detector stops working, it says "labs processing" on the screen. Yes, you can manage the parameters without problems, even re-read the QR code

**Comment by dnewman-gpsw** [comment]
Try !2R or !3R. The camera is not crashing, the likely error is it taken too long to close the last file before beginning a new detection.  This will be fixed in the next update, but increasing the delay should help the current firmware.

**Reply by Ezequiellpi** [reply]
I use the Android application to generate the QR. How can I modify the command manually? What exactly does that command do?

**Reply by Ezequiellpi** [reply]
I'm already trying it!

**Reply by Ezequiellpi** [reply]
thanks a lot! It's been running for over an hour! I'll keep trying.

**Reply by dnewman-gpsw** [reply]
Type it is manually to the Added Command field

**Comment by damikun** [comment]
Hi also im facing this issue..( Hero 12 on 2 cameras)  Looking forward to fix it in next labs release..
